proto/cosmos/crypto/keyring/types.proto

```diff
@@ -0,0 +1,69 @@
+syntax = "proto3";
+package cosmos.crypto.keyring;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/cosmos/cosmos-sdk/crypto/keyring";
+option (gogoproto.goproto_getters_all) = false;
+
+// TODO - define proto structs for all structs in cryto/kerying/info.go and remove the original structs.
+
+// Item is a keyring item stored in a keyring backend.
+message LocalInfo {
+  option (gogoproto.goproto_stringer) = false;
+
+  string name = 1;
+  bytes key = 2 [(gogoproto.casttype) = "crypto/ed25519.PublicKey"];
+  string privKeyArmor = 3;
+  PubKeyType algo = 4;
+}
+
+enum PubKeyType{
+  MultiType = 0;
+  Secp256k1Type = 1;
+  Ed25519Type = 2;
+  Sr25519Type = 3;
+}
+
+message LedgerInfo {
+  option (gogoproto.goproto_stringer) = false;
+
+  string name = 1;
+  bytes key = 2 [(gogoproto.casttype) = "crypto/ed25519.PublicKey"];
```

Comment:
`Any pub_key` (or maybe we can hardcode secp256k1 here?)

Reply:
I am not sure about secp256k1, should we introduce other curves for pub key or only secp256k1?

Reply:
What Amaury was saying was to use Any - with that you will handle all types which implement `PubKey` interface.
I have other PR to add support for `secp256r1`, which will be merged after this PR.